docs/02_cleos/03_command-reference/net/disconnect.md

```diff
@@ -1,17 +1,56 @@
+## Command
+```sh
+cleos net disconnect [OPTIONS] host
+```
+
+**Where:**
+* [OPTIONS] = See **Options** in the [**Command Usage**](command-usage) section below.
+* host = The hostname:port to disconnect from.
+
+**Note:** The arguments and options enclosed in square brackets are optional.
+
 ## Description
-close an existing connection
+Close an existing connection to a specified peer. This command allows an operator's node to disconnect from another peer without restarting the node.
+
+## Command Usage
+The following information shows the different positionals and options you can use with the `cleos net disconnect` command:
+
+### Positionals
+* `host` _TEXT_ REQUIRED - The hostname:port to disconnect from.
+
+### Options
+* `-h,--help` - Print this help message and exit.
```

Comment:
Full stop at end ... template indicate this list doesn't have them

Reply:
Removed.